cron/hourly/git_maintenance_tasks_lock

```diff
@@ -4,4 +4,5 @@ set -e
 # Run git_maintenance_tasks only if not running already
 DIR="$(readlink -e "$(dirname $0)/..")"
 mkdir -p ~/lmms-repo
-flock -w 0 ~/lmms-repo/git_maintenance_tasks.lock "$DIR/git_maintenance_tasks"
+flock -w 0 ~/lmms-repo/git_maintenance_tasks.lock \
+	"$DIR/git_maintenance_tasks_retry"
```

Comment:
>Can't we do this and save the redundant script?

Not like that, the new instance would mess with the running one. You could use the `-c` flag, but this is less flexible; you have discarded the error message. A new retry strategy (e.g., running `git reset`) could be added to the script if these errors are frequent.